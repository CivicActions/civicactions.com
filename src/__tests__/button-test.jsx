import React from "react";
import TestRenderer from "react-test-renderer";

import Button from "../components/atoms/Buttons";


const testRenderer = TestRenderer.create(
  <Button
    button_text = "Button text"
    link = "http://www.civicactions.com"
    isExternal = { true }
  />
);




describe("Button", () =>
  it("renders button text", () => {
    const testInstance = testRenderer.root;
    expect(testInstance.findByType(Button).props.button_text).toBe('Button text');
  }))
