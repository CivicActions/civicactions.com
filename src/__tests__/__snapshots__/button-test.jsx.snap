// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button snapshot test. renders button text 1`] = `
<button
  className="usa-button"
>
  <a
    href="http://www.civicactions.com"
  >
    Button text
  </a>
</button>
`;

exports[`Button snapsjhot test. renders button text 1`] = `
<button
  className="usa-button"
>
  <a
    href="http://www.civicactions.com"
  >
    Button text
  </a>
</button>
`;
